Stripe/PublicHeaders/STPAddress.h

```diff
@@ -220,7 +220,10 @@ extern STPContactField const STPContactFieldName;
  @return The closest representation of the billing address requirement as
  a PKAddressField value.
  */
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated"
```

Comment:
Looks like this should go before the documentation comment, build is failing because jazzy thinks these two methods are undocumented.

Reply:
👍 fixed in da1cbea